modules/preview-react/select/lib/SelectBase.tsx

```diff
@@ -49,6 +49,10 @@ export interface NormalizedOption extends Option {
   label: string;
 }
 
+export interface RenderSelectedOptionFunction {
+  (option: NormalizedOption): React.ReactNode;
```

Comment:
I opted to use `NormalizedOption` instead of `RenderableOption` because the selected property is redundant and I thought the focused property could be confusing since it'll apply to an option that isn't rendered in the menu list

Reply:
Makes sense 👍